harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -788,11 +1015,38 @@ def _load(self, load_path: pathlib.Path) -> None:
                     "callback will be initialized from scratch"
                 )
 
-        # Load workload sequencer state.
-        wlsq_path = load_path.joinpath("workload_sequencer.pkl")
-        if self.wlsq is not None and wlsq_path.exists():
-            with wlsq_path.open("rb") as f:
-                self.wlsq.load_state(pickle.load(f))
+        save_path = load_path.joinpath("trial_state.pkl")
+        if save_path.exists():
+            with save_path.open("rb") as f:
+                self.load_state(pickle.load(f))
+        else:
+            # Support legacy save states
+            wlsq_path = load_path.joinpath("workload_sequencer.pkl")
+            if wlsq_path.exists():
+                with wlsq_path.open("rb") as f:
+                    self.load_state(pickle.load(f))
```

Comment:
Ambitious!  I was kinda expecting you to tell me "that's crazytalk we'll just tell those users sorry"